CMakeLists.txt

```diff
@@ -66,14 +71,15 @@ install(TARGETS pHash DESTINATION lib)
 
 set(EXAMPLEDIR examples)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
 if(HAVE_IMAGE_HASH)
 add_executable(TestDCT "${EXAMPLEDIR}/test_imagephash.cpp")
 target_link_libraries (TestDCT pHash)
 install(TARGETS TestDCT DESTINATION bin)
 
-add_executable(TestRadish "${EXAMPLEDIR}/test_radish.cpp")
-target_link_libraries (TestRadish pHash)
-install(TARGETS TestRadish DESTINATION bin)
+#add_executable(TestRadish "${EXAMPLEDIR}/test_radish.cpp")
```

Comment:
Please uncomment the test after adding the test_radish.cpp back into the project

Reply:
Seems like you did this already, so I dropped the patch